apps/web/src/components/Home/Timeline/index.tsx

```diff
@@ -48,6 +48,7 @@ const Timeline: FC = () => {
   });
 
   const publications = data?.feed?.items;
+  console.log('publications lenght', publications?.length);
```

Comment:
To remove :)

Reply:
ah damn, some debugging from another branch 🤣